core/plugins/org.polarsys.capella.core.platform.sirius.sirius.validation/src/org/polarsys/capella/core/platform/sirius/sirius/validation/parser/helper/DescriptionParserHelper.java

```diff
@@ -37,4 +41,12 @@ public static List<String> getLinksWithId(String description, String linkId) {
     return links;
 
   }
+  
+  public static String getElementName(EObject element) {
+    if (element instanceof ReNamedElement) {
```

Comment:
Might be good to move it directly to NamingHelper
https://github.com/eclipse/capella/pull/2628/files, will see if there is cycle issues

Reply:
yeah that was the reason it was moved there :p